index.md

```diff
@@ -5,20 +5,14 @@ The Academie d'Espee exists to promote the practice and enjoyment of period styl
 
 ## Structure of the Academie d'Espee
 
-* [General Structure Information](articles/guild)
-* Masters
-  * [Roster of Masters](masters)
-  * [Legacy of Collars](collars)
-* Provosts
-  * [What is a Provost](articles/guidelines)
-  * [Roster of Provosts](provosts)
-* Free Scholars
-  * [What is a Free Scholar](articles/freescholar-2)
-  * [Roster of Free Scholars](fs)
-* Scholars
-  * [What is a Scholar](articles/scholar)
-  * [Roster of Scholars](scholars)
+* [Guild Charter](charter)
+* [Bylaws](bylaws)
```

Comment:
With these on the main menu, is it needed here?

Reply:
Unsure.  Index page feels strangely empty without it.  Will show the page to the rest of the council.